sql/2025/05/featured-image-block-usage.sql

```diff
@@ -0,0 +1,73 @@
+# HTTP Archive query to get % WordPress sites using Featured Image in block themes.
+#
+# WPP Research, Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# See query results here: https://github.com/GoogleChromeLabs/wpp-research/pull/184
+
+DECLARE
+  DATE_TO_QUERY DATE DEFAULT '2025-03-01';
```

Comment:
```suggestion
  DATE_TO_QUERY DATE DEFAULT '2025-04-01';
```

Let's use the latest dataset

Reply:
Thanks! When I created this PR earlier this month, I believed that dataset wasn't available for querying. But since it's now available, it makes more sense to use the latest version.